.github/workflows/ci.yml

```diff
@@ -0,0 +1,69 @@
+name: Compile and run tests
+
+on: [push, pull_request]
+
+jobs:
+  test_standard_build:
```

Comment:
The actual job runs in Fedora even though the test runner is Ubuntu.

This could probably attempt to build the RPM instead -- I just didn't try to do that.